content/en/css.md

```diff
@@ -37,11 +37,15 @@ Use the `<link>` tag to include all your stylesheets in the `<head>` of the docu
 
 Do not put styling information into your HTML markup directly, either with the `style` attribute that accepts CSS or with deprecated attributes such as `align`, `border`, or `width`. These are difficult to maintain and make it harder to track down what is causing an element to appear as it does.
 
+##### Caveat
+
+In some cases, it is considered good practice to inline critical styles in a `style` block in the document's `head`. In this case critical inline styles are identified either manually or through the use of a tool. Non critical styles can then be asyncronously loaded, increasing the perceived page load speed.
+
 #### Formatting
 
 Put each selector on its own line and each property on its own line for easy readability and so version control systems can clearly show which parts have changed. The attributes within a selector should be alphabetized for easy scanning and so that compression algorithms like gzip have a greater chance of finding repeatable patterns.
 
-```css
+```
```

Comment:
Prism needs "markup" or "css" next to the backticks so the correct CSS class is put on the code block, so the syntax highlighting will be appropriate to the language.